Get business alljobs

jinsim95 opened the pull request:
Gets the list of all jobs that the business has, returns an array of those jobs which is then used to populate the business dashboard (.../business/dashboard).

ando-huang reviewed (commented):
niceee

mattfan00 commented on back-end/routes/getBusinessJobs.js:
```diff
@@ -0,0 +1,23 @@
+    }
+  ]
+  
+  res.json(user);
```
here, `res.json` should have `alljobs` as the parameter

mattfan00 reviewed (commented):
Hey Jin, this looks good! I tried testing the code on my end and saw that the homepage does not seem to be rendering the contents of the `/business/alljobs` route's response. I highlighted one of the areas, which is that `res.json` does not seem to have the correct variable as the parameter. Another change I think you will have to make is "adding the route" to `app.js`. Currently in `app.js`, there is an example route that is added by using `require` to import the route file from the `./routes` directory, and then calling `app.use` to make that route available for use. I think after those changes you will be able to see the contents of the response in the console from the frontend

jinsim95 commented:
@mattfan00 Thanks for letting me know! I had implemented those previously but I think there was an issue with git commit and the changes didn't get reflected correctly. I fixed it by using "git commit -a ..." instead and I think it worked this time. Let me know if there seems to be any other issues!

mattfan00 commented:
> @mattfan00 Thanks for letting me know! I had implemented those previously but I think there was an issue with git commit and the changes didn't get reflected correctly. I fixed it by using "git commit -a ..." instead and I think it worked this time. Let me know if there seems to be any other issues!

yayyy this looks great, thanks Jin!
